src/astro/databases/base.py

```diff
@@ -51,6 +52,14 @@ def sqlalchemy_engine(self) -> sqlalchemy.engine.base.Engine:
         """Return Sqlalchemy engine."""
         return self.hook.get_sqlalchemy_engine()
 
+    def get_sqlalchemy_engine(self, table: Table):
+        return self.hook.get_sqlalchemy_engine()
```

Comment:
ATM, this implementation is exactly the same as the property `sqlalchemy_engine` on line 51.